src/Components/test/testassets/BasicTestApp/CounterComponent.cshtml

```diff
@@ -1,6 +1,6 @@
 <h1>Counter</h1>
 <p>Current count: @currentCount</p>
-<p><button onclick="@((handleClicks ? (Action)IncrementCount : null))">Click me</button></p>
+<p><button onclick="@((handleClicks ? (Action)IncrementCount : () => { }))">Click me</button></p>
```

Comment:
@SteveSandersonMS - something else to discuss here - this was failing because `EventCallback.Factory.Create` doesn't accept a null delegate. What do you think should be the behaviour of a null-event-handler, should it trigger `HandleEventAsync`?

Reply:
Hmm, I'd imagine it would be common to supply a null value for an event handler if you're using delegate types instead of `EventCallback`, e.g.:

```
<button onclick=@DoSomething>Click me</button>
@functions {
    [Parameter] Action DoSomething { get; set; }
}
```

In this case, it's inconvenient to deal with absence of a value for the `DoSomething` parameter (you'd either wrap `DoSomething` in your own method, or use `onclick=@(() => DoSomething?.Invoke())`, which is pretty laborious and nonobvious to non-C#-experts). Previously it was a no-op if the value was null, but now it's an error. I think the no-op is maybe more useful.

Do you think it's enough to tell people "remember to use `EventCallback`, or if you don't, you are responsible for null checking"?

Reply:
Yeah that's what I wondered as well. I'll fix this and add some tests.